### DISCUSSION TITLE
5.6.0

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#1041] [🚀 Releases]
## What's Changed
- Custom `@Generated` annotation #1016 #1036
  - New config to specify custom @Generated annotation: `generatedAnnotation`
  - Modified the logic of setting a default value for `@Generated` annotation (**if value for `generatedAnnotation` config is empty**). Following classes will be looked (up in the same order): `@jakarta.annotation.Generated` -> `@javax.annotation.processing.Generated` -> `@javax.annotation.Generated`. If none are present in the classpath then `@Generated` annotation will not be added on top of the classes.
- New config to generate only no-args constructor: `generateNoArgsConstructorOnly` #1017 #1037
- Generate Jackson annotations in interfaces #1033 #1034
- Ability to combine multiple response projections #985 #1031
  - Generating a new constructor that accepts a list of response projections of the same type. Resulting response projection will contain a merge of all fields of the provided projections.
- Fix generation of interfaces when return type of the method has a list of Unions #1018 #1030
- Kotlin:
  - Added support for annotations in generated kotlin operations #1002 #1003
  - Save kotlin nullable sign for custom type mapping and apiReturnType #1019 #1020
- SBT: 
  - Update dependencies of SBT plugin #1001

**Full Changelog**: https://github.com/kobylynskyi/graphql-java-codegen/compare/v5.5.0...v5.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.6.0'>5.6.0</a>.</em>

**Comment by kobylynskyi** [comment]
SBT plugin release

**Reply by kobylynskyi** [reply]
@jxnu-liguobin could you please release SBT plugin?

**Reply by jxnu-liguobin** [reply]
yes, and update some
